api/HTMLSourceElement.json

```diff
@@ -141,7 +141,8 @@
             "opera": "mirror",
             "opera_android": "mirror",
             "safari": {
-              "version_added": "10.1"
+              "version_added": "10.1",
+              "notes": "The <a href='https://developer.mozilla.org/docs/Web/HTML/Element/source#attr-sizes'><code>sizes</code> attribute is supported since Safari 9.1."
```

Comment:
```suggestion
              "notes": "The <a href='https://developer.mozilla.org/docs/Web/HTML/Element/source#attr-sizes'><code>sizes</code></a> attribute is supported since Safari 9.1."
```